app/assets/stylesheets/styleguide/_typography.scss

```diff
@@ -1,218 +0,0 @@
-@import "_print.scss";
-
```

Comment:
Are we sure these mixins were not being used? Just thinking these could be out of date and be used, so us removing them could be a regression.